x-pack/plugins/security_solution/public/management/services/policies/hooks.ts

```diff
@@ -40,3 +45,34 @@ export function useGetEndpointSpecificPolicies(
     }
   );
 }
+
+export function useGetAgentCountForPolicy({
+  onError,
+  policyIds,
+}: {
+  onError?: (error: ServerApiError) => void;
+  policyIds: string[];
+}): QueryObserverResult<GetAgentPoliciesResponse> {
+  // is there a more elegant way to do this? w/o the conditional, the call is a bad request when the policyId is empty
```

Comment:
anyone have thoughts on this or is this the clear enough?

Reply:
We should make sure we bump the page size on this call to `50` that is the make amount of Policies we could be displaying on a single page.  This will ensure we get all of the Agent Policies.

I'm seeing this when testing
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/56395104/157332005-cb52060f-b4c2-4c5f-bf64-9729875f9cf8.png">